File: CS/WinSolution.Module.Win/Updater.cs
using System;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;
using DevExpress.Persistent.BaseImpl;

namespace WinSolution.Module.Win {
    public class Updater : ModuleUpdater {
        public Updater(IObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            Person p = ObjectSpace.CreateObject<Person>();
            p.FirstName = "Test";
            PhoneNumber pn = ObjectSpace.CreateObject<PhoneNumber>();
            pn.Party = p;
            pn.Number = "123";
        }
    }
}

File: VB/WinSolution.Module.Win/Updater.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Persistent.BaseImpl

Namespace WinSolution.Module.Win
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal objectSpace As IObjectSpace, ByVal currentDBVersion As Version)
			MyBase.New(objectSpace, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			Dim p As Person = ObjectSpace.CreateObject(Of Person)()
			p.FirstName = "Test"
			Dim pn As PhoneNumber = ObjectSpace.CreateObject(Of PhoneNumber)()
			pn.Party = p
			pn.Number = "123"
		End Sub
	End Class
End Namespace
